.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Default owner for everything in this repository
+*   @mozilla-services/obs-team
```

Comment:
This looks fine to me; FWICT the extra whitespace doesn't matter (file is valid; I see similar extra whitespace in the [example file](https://github.com/mozilla/fxa/blob/ed0de1b49ac1c548c8dd3b2d299c62ab1b6d2897/.github/CODEOWNERS#L1) in the docs).